api/src/constants.ts

```diff
@@ -31,3 +31,8 @@ export const projectSetNames = [
   'test',
   'prod',
 ];
+
+export const ROLE_ID = {
+  PRODUCT_OWNER: 1,
+  TECHNICAL_STEWART: 2,
```

Comment:
TECHNICAL_STEWART is used as the constant, however Technical contact is used everywhere else. Not sure if one has been decided as the official title?